test_user_management.py

```
@@ -0,0 +1,391 @@
+import hashlib
+import os
+import json
+from typing import Dict, List, Optional
+import logging
+from datetime import datetime, timedelta
+import re
+
+# SECURITY ISSUE 1: Hardcoded database credentials and connection string
+DATABASE_URL = "sqlite:///users.db"
+ADMIN_USERNAME = "admin"
+ADMIN_PASSWORD = "admin123"  # SECURITY ISSUE: Hardcoded weak password
```

Ruff (0.15.6)

[error] Possible hardcoded password assigned to: "ADMIN_PASSWORD"

(S105)